examples/simple-game-server/Makefile

```diff
@@ -82,9 +82,13 @@ endif
 ifeq ($(WITH_ARM64), 1)
 push: push-linux-image-arm64
 endif
-	-DOCKER_CLI_EXPERIMENTAL=enabled docker manifest rm $(server_tag)
-	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create $(server_tag) $(push_server_manifest)
-	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest push $(server_tag) --purge
+	-docker manifest rm $(server_tag)
+	docker manifest create $(server_tag) $(push_server_manifest)
+	docker manifest push $(server_tag) --purge
+	# TODO: Update to replace `docker manifest` commands if/when the above commands fail (~Docker 24.x)
```

Comment:
The above commands are still marked as "experimental" but work in Docker 20.x (which is what we have locally and on Cloud Build).  `docker buildx imagetools create` has a bug in it with GCR/Artifact registry though.

I tested on Docker 24.x on a different machine where `docker manifest` now breaks (it is experimental) but `docker buildx imagetools create` works. So leaving it for when the versions eventually get updated and we'll need a reference on how to fix things.